src/replication_misc.c

```diff
@@ -310,3 +312,19 @@ set_socket_keepalive(int sfd)
 out:
 	return (ret);
 }
+
+json_object *
+json_object_new_uint64(uint64_t value)
+{
+	/* 18446744073709551615 */
+	char num[21];
+	int r;
+	json_object *jobj;
+
+	r = snprintf(num, sizeof(num), "%" PRIu64, value);
+	if (r < 0 || (size_t)r >= sizeof(num))
+		return NULL;
```

Comment:
wrap the "0" into json_object_new_string instead of returning NULL.

Reply:
Here, returning `NULL` will give value as `null` (json_object_type will be `json_type_null`).

Reply:
@utkarshmani1997 if the value is `null` then...
- It will be treated as 0 in GO if it is defined as `int/float64`.
- It will be null for a string.
I executed the same program mentioned in that link and it didn't throw any error..!

Reply:
Thanks, i was also unaware of this